### DISCUSSION TITLE
RMC DÉCOUVERTE and RMC STORY

### DISCUSSION THREAD

**Discussion by martynojunk** [iptv-org/community#996] [Q&A]
Hello, is it possible to add RMC DÉCOUVERTE and RMC STORY. They are 2 french public channel's. 
Thank you

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/fr.m3u#L206-L209
